build.gradle

```diff
@@ -27,6 +27,14 @@ deploy {
                 // getTargetTypeClass is a shortcut to get the class type using a string
 
                 frcJava(getArtifactTypeClass('FRCJavaArtifact')) {
+                 // Launch VisualVM with --jdkhome $PATH_TO_WPI_JDK
```

Comment:
Have you verified that the profiler even works on the rio Jre?

Reply:
yes it works